haas.wsgi

```diff
@@ -1,7 +1,14 @@
 #!/usr/bin/env python
-import haas.api
-from haas import config, model, server, migrations
+from haas import api, config, server, migrations
 config.setup('/etc/haas.cfg')
 server.init()
 migrations.check_db_schema()
 from haas.rest import app as application  # noqa
+
+# `api` is imported for the side-effect of registering the api functions.
+# silence a pylint error about an unused import:
+api
+
+# ...and the app is imported because this is a wsgi script, and its job
+# is to define `application`. Again, silence the error:
+application
```

Comment:
Why not use `# pylint: disable=unused-import` or `# pylint: disable=W0611` on each offending line to silence the errors? Makes everything a little more self-documenting, and a little less ugly (in my opinion)